kyuubi-main/src/test/scala/org/apache/kyuubi/operation/KyuubiOperationPerConnectionSuite.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.kyuubi.config.KyuubiConf
 /**
  * UT with Connection level engine shared cost much time, only run basic jdbc tests.
  */
-class KyuubiOperationPerConnectionSuite extends WithKyuubiServer with BasicJDBCTests {
+class KyuubiOperationPerConnectionSuite extends WithKyuubiServer with JDBCTestUtils {
```

Comment:
We don't need to test basic tests for connection share level ?

Reply:
The cases in `JDBCTestUtils` got tested on Engine side, and several times on Kyuubi Server side, it's quite redundant. Thus just keep connection share level specific cases in `KyuubiOperationPerConnectionSuite` to simplify the test suites.

Reply:
make sense